manim/mobject/types/vectorized_mobject.py

```diff
@@ -264,36 +268,37 @@ def fade(self, darkness=0.5, family=True):
         super().fade(darkness, family)
         return self
 
-    def get_fill_rgbas(self):
+    def get_fill_rgbas(self) -> np.ndarray:
         try:
             return self.fill_rgbas
         except AttributeError:
             return np.zeros((1, 4))
 
-    def get_fill_color(self):
+    def get_fill_color(self) -> Color:
         """
         If there are multiple colors (for gradient)
         this returns the first one
         """
         return self.get_fill_colors()[0]
 
-    def get_fill_opacity(self):
+    def get_fill_opacity(self) -> float:
         """
         If there are multiple opacities, this returns the
         first
         """
         return self.get_fill_opacities()[0]
 
-    def get_fill_colors(self):
+    def get_fill_colors(self) -> Sequence[Color]:
```

Comment:
Return types should be as specific as possible
```suggestion
    def get_fill_colors(self) -> List[Color]:
```